sdk/python/responsible-ai/mlflow-deployment-with-explanations/mlflow-deployment-with-explanations.ipynb

```diff
@@ -0,0 +1,904 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_file_path = (\n",
+    "    f\"./artifacts/original_model/{model_name}/<FILE_PATH_TO_MLFLOW_FOLDER>\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.2 Get original model signature"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If there is one, load the model signature from Azure ML."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mlflow.models import Model\n",
+    "\n",
+    "mlflow_model = Model.load(model_uri)\n",
+    "model_signature = mlflow_model.signature\n",
+    "print(f\"Model Signature: {model_signature}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.3 Register wrapper model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Save and then register the model to your workspace. If you are re-running this step, you may need to delete the folder containing the last version of the saved wrapper model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_file_path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "artifacts = {\"model\": model_file_path, \"RAI insights\": \"./artifacts/RAI_Insights\"}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mlflow.pyfunc.save_model(\n",
+    "    path=f\"artifacts/{wrapper_model_name}\",\n",
+    "    code_path=[\"./explanation_wrapper.py\"],\n",
+    "    conda_env=\"./env.yml\",\n",
+    "    python_model=explanation_wrapper,\n",
+    "    artifacts=artifacts,\n",
+    "    signature=model_signature,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mlflow.register_model(f\"file://artifacts/{wrapper_model_name}\", wrapper_model_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Deploy Wrapper Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.1 Deployment Information **(user input required)**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Enter the name of an **existing** endpoint and the deployment name of your choice."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint_name = \"<AML_ENDPOINT_NAME>\"\n",
+    "deployment_name = \"<DEPLOYMENT_NAME>\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2 Get wrapper model ID"
```

Comment:
I would say "the new model ID" wrapper model ID otherwise sounds like it is a thing. Ignore this if it doesn't sound like that to you!